ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/DirectionsQuestionnaireAssert.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.ccd.assertion;
+
+import org.assertj.core.api.AbstractAssert;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.DirectionsQuestionnaire;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class DirectionsQuestionnaireAssert extends AbstractAssert<DirectionsQuestionnaireAssert, DirectionsQuestionnaire> {
+    public DirectionsQuestionnaireAssert(DirectionsQuestionnaire directionsQuestionnaire) {
+        super(directionsQuestionnaire, DirectionsQuestionnaire.class);
+    }
+
+    public DirectionsQuestionnaireAssert isEqualTo(DirectionsQuestionnaire directionsQuestionnaire) {
+
+        assertThat(actual.getExceptionalCircumstancesReason()).isEqualTo(directionsQuestionnaire.getExceptionalCircumstancesReason());
```

Comment:
why are we only asserting on `getExceptionalCircumstancesReason`? What about the rest of the fields?

Reply:
i was still working on tests at that point but now it's complete..